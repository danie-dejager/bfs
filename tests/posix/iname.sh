bfs_diff basic -iname '*F*'
