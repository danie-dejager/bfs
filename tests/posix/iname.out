basic/e/f
basic/j/foo
basic/k/foo
basic/l/foo
